Resources/public/js/views/services/plugins/ez-objectrelationloadplugin.js

```diff
@@ -36,11 +36,12 @@ YUI.add('ez-objectrelationloadplugin', function (Y) {
         _loadFieldRelatedContent: function (e) {
             var loadOptions = {api: this.get('host').get('capi')},
                 relatedContent = this.get('relatedContent'),
-                contentDestination = this.get('host').get('content').relations(
-                    'ATTRIBUTE', e.fieldDefinitionIdentifier
-                ).shift();
+                fields = this.get('host').get('content').get('fields'),
+                destinationContentId = fields[e.fieldDefinitionIdentifier].fieldValue.destinationContentId,
```

Comment:
you are kind of duplicating the `relations` method here, so you should still use it (that means `relations` has to be able to handle both `ezobjectrelation` and `ezobjectrelationlist` fields)